server/seeder/modules/fileComposer/fileComposer.js

```diff
@@ -51,6 +51,17 @@ export default class FileComposer {
     this.#orderObject();
   }
 
+  unitTest(method) {
```

Comment:
OK I see the trick here.
It works well, but I don't know if it's the better way to implement it. We'll have to list the pros/cons of this approach.

For me.
Pros:
- We don't have to write consumers that instantiate classes for test simulation
- I like the syntax it offers in the tests (e.g. `fileComposer.unitTest('getDeepTree')).toThrow();`)

Cons:
- We attach a function that deals with tests in the class. I'd tend to leave this to .spec files
- The switch will grow as the number of testable methods in the class grows (With my understanding Jest use a factory pattern to deal with that)

---

I'll read again reference https://jestjs.io/docs/es6-class-mocks, maybe I missed something when I added the first dummy tests.